chart/infra-server/static/flavors.yaml

```diff
@@ -1024,6 +1024,82 @@
     - name: cluster-console-password
       description: The password to login at the openshift console
 
+#########################
+#  Openshift IBM ROKS   #
+#########################
+- id: ibmroks
+  name: IBMROKS
+  description: IBM ROKS cluster
+  availability: stable
+  workflow: configuration/workflow-openshift-ibmroks.yaml
+  parameters:
+    - name: name
+      description: cluster name
+      value: example1
+      help: The name must start with a letter, can contain letters, 
+        numbers, periods (.), and hyphen (-), and must be 35 
+        characters or fewer. Use a name that is unique across 
+        regions. The cluster name and the region in which the 
+        cluster is deployed form the fully qualified domain name 
+        for the Ingress subdomain. To ensure that the Ingress 
+        subdomain is unique within a region, the cluster name 
+        might be truncated and appended with a random value within 
+        the Ingress domain name.
+        https://cloud.ibm.com/docs/openshift?topic=openshift-cluster-create-vpc-gen2&interface=cli
+
+    - name: nodes
+      description: number of nodes
+      value: "2"
+      kind: optional
+
+    - name: instance-type
+      description: node machine type
+      value: cx2.8x16
+      kind: optional
+
+    - name: zone
+      description: IBM Cloud zone to deploy into.
```

Comment:
Would be helpful to have a (non-exhaustive) list of possible zones here in the help field.